website/models.py

```diff
@@ -7,4 +7,24 @@ class User(models.Model):
     last_name = models.CharField(max_length=64)
     email_address = models.CharField(max_length=254)
     password = models.CharField(max_length=64)
-    salt = models.CharField(max_length=64)
\ No newline at end of file
+    salt = models.CharField(max_length=64)
+
+
+class Notification(models.Model):
+    id = models.CharField(max_length=64)
```

Comment:
You will want to add a user field to the notification so that they are related.

That would look like 

user = models.ForeignKey(User)

That implies that each Notification has a user, but each user can have multiple notifications.